### PR TITLE
Added quotes around sourcePath to handle spaces in path.

### LINKED ISSUE #158: git clone fails if there are spaces in path
My username on my windows machine has a space in it and the git clone commands were failing because there were no quotes around the path so the command line thought the path was multiple arguments.

### PR DESCRIPTION
**Pull request by marler8997**
fixes #158

### CODE REVIEW

File: Protobuild.Internal/Packages/PackageResolve/SourcePackageResolve.cs [hunk 1]
```diff
@@ -163,7 +163,7 @@ private string GetSourcePackage(string url)
             }
 
             Directory.CreateDirectory(sourcePath);
-            GitUtils.RunGit(null, "clone --progress --bare " + url + " " + sourcePath);
+            GitUtils.RunGit(null, "clone --progress --bare " + url + " \"" + sourcePath + \");
```
Comment (hach-que) [inline Protobuild.Internal/Packages/PackageResolve/SourcePackageResolve.cs]: This doesn't look right; the \" should have quotes around it.

**Comment by hach-que** [comment]
Should this logic escape quotes within the path as well?

**Comment by marler8997** [comment]
In Windows and Linux, filenames and directories can't contain quotes.  Since this argument is a path/filename, I don't think there's a case where it could contain quotes that would need to be escaped.

File: Protobuild.Internal/Packages/PackageResolve/SourcePackageResolve.cs (hunk 1 above)
Comment (marler8997) [inline Protobuild.Internal/Packages/PackageResolve/SourcePackageResolve.cs]: Ya woops!  Fixed

**Comment by hach-que** [comment]
Thanks for this fix.  I have it working on my machine.

At some point in the near future, I might need you to agree to the CLA on https://cla.dotnetfoundation.org/.  Are you okay with this and do you agree to the terms of the CLA as laid out on that website?

**Comment by marler8997** [comment]
Yes I'm fine with it and I agree to the CLA terms.

**Comment by hach-que** [comment]
Thanks, I've merged this change in :)

(Due to the way GitHub works, it doesn't detect my amend-and-push as a merge, so I'll close it here)

**Pull request closed without merging**